gamestonk_terminal/prediction_techniques/README.md

```diff
@@ -273,3 +268,24 @@ Long_Short_Term_Memory \
 
 <img width="992" alt="Captura de ecrã 2021-03-14, às 00 06 51" src="https://user-images.githubusercontent.com/25267873/111053158-420c7300-8459-11eb-993b-6d9c26f98af9.png">
 
+## conv1d <a name="conv1d"></a>
+```
+usage: lstm [-d N_DAYS] [-i N_INPUTS] [-j N_JUMPS] [--epochs N_EPOCHS] [-p {normalization,standardization,minmax,none}]
```

Comment:
usage: conv1d AND jump can be removed

Reply:
If epochs, preprocessing, lossfunction and learning rate end up being set on config_neural_network. Then we can remove them from all the functions here.